src/annotator/test/annotation-sync-test.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+
+var AnnotationSync = require('../annotation-sync');
+var EventEmitter = require('tiny-emitter');
```

Comment:
Third-party imports should be at the top, followed by a blank line, followed by local imports.